source/Cosmos.System2/Graphics/SVGAIICanvas.cs

```diff
@@ -419,14 +419,14 @@ public override void Display()
         /// </summary>
         /// <param name="str">string to draw.</param>
         /// <param name="aFont">Font used.</param>
-        /// <param name="pen">Color.</param>
+        /// <param name="color">Color.</param>
         /// <param name="x">X coordinate.</param>
         /// <param name="y">Y coordinate.</param>
-        public override void DrawString(string str, Font aFont, Pen pen, int x, int y)
+        public override void DrawString(string str, Font aFont, Color color, int x, int y)
```

Comment:
While this method signature was like this from the start, can we change all the parameters to adhere to the naming convention defined by the .editorconfig? That way, if we ever will want to change it in the future, we can use a refactoring tool.

Reply:
what do you mean?

Reply:
When following the settings defined in `.editorconfig`, every parameter name should be prefixed with "a". I guess this doesn't really matter here, so on second thought, this can be ignored XD